uktrade/data-hub-frontend
# Migrate edit client relationship manager page to use new layout

## Description of change

The page for editing the client relationship manager for an investment project has been updated to use the new project layout. This page has also been moved into the `modules` directory and is using React Router.

## Test instructions

Go to an investment project page and edit the client relationship manager (accessible from the project team page). Everything should work as expected.

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
